src/js/edu-benefits/utils/validations.js

```diff
@@ -303,8 +303,8 @@ function isValidBenefitsHistoryPage(data) {
 }
 
 function isValidRotcScholarshipAmount(data) {
-  return isValidField(isValidMonetaryValue, data.amount)
-    && isValidField(isValidYear, data.year);
+  return (isBlank(data.amount.value) || isValidField(isValidMonetaryValue, data.amount))
+    && (isBlank(data.amount.value) || isValidField(isValidYear, data.year));
```

Comment:
Yep, I'll fix that.

For the second question, I don't think we have a good way of preventing that. It makes the growable table even more complicated. It's something we should probably talk about for the future, though.